internal/configs/version1/template_helper.go

```diff
@@ -5,18 +5,17 @@ import (
 	"text/template"
 )
 
-// SplitInput splits the input from "," and returns an array of strings
-func SplitInput(s string, delim string) []string {
+// splitinput splits the input from "," and returns an array of strings
+func splitinput(s string, delim string) []string {
```

Comment:
For multiword names, the convention is to use mixed caps. Like "SplitInput". See https://golang.org/doc/effective_go.html#mixed-caps
At the same time, as mentioned here https://github.com/nginxinc/kubernetes-ingress/pull/691#discussion_r323989004, I suggest renaming to `split`. This will make the name simpler, as the suffix input seems unnecessary. 
Same applies for `triminput`.